File: 3536-maximum-product-of-two-digits/3536-maximum-product-of-two-digits.cpp
class Solution {
public:
    int maxProduct(int n) {
        vector<int>ans;
        while(n!=0){
            int digit=n%10;
            ans.push_back(digit);
            n=n/10;
        }
          sort(ans.begin(),ans.end());
          return ans[ans.size()-1]*ans[ans.size()-2];
    }
};

File: 3536-maximum-product-of-two-digits/3536-maximum-product-of-two-digits.java
class Solution {
    public int maxProduct(int n) {
       ArrayList<Integer>ans=new ArrayList<>();
        while(n!=0){
            int digit=n%10;
            ans.add(digit);
            n=n/10;
        }
        Collections.sort(ans);
        return ans.get(ans.size()-1)*ans.get(ans.size()-2);
        
    }
}